### PR TITLE
Updated the rulesets

### PR DESCRIPTION
**Pull request by ChathuminaVimukthi**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] I have read the **CONTRIBUTING** document.
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by shalithasuranga** [comment]
@ChathuminaVimukthi  Could you provide reference links for added rules 

Thanks

**Comment by ChathuminaVimukthi** [comment]
https://www.tutorialspoint.com/html5/html5_deprecated_tags.htm
https://www.sitepoint.com/5-obsolete-features-html5/
https://www.w3.org/TR/html5/

File: src/rulesets/obsoleteTags.js [hunk 1]
```diff
@@ -50,4 +50,10 @@ module.exports = [
         error : '<listing> is completely obsolete, and must not be used by authors.',
         solution : 'Use <pre> and <code> instead.'
     },
+    {
+        tag : 'center',
```
Comment (shalithasuranga) [inline src/rulesets/obsoleteTags.js:54]: This is ok.. Thanks

File: src/rulesets/obsoleteAttr.js [hunk 2]
```diff
@@ -60,4 +60,10 @@ module.exports = [
         error : 'align attribute in <h1>...<h6> is completely obsolete, and must not be used by authors.',
         solution : 'Use CSS instead.'
     },
+    {
+        tags : 'time',
```
Comment (shalithasuranga) [inline src/rulesets/obsoleteAttr.js]: Can you please show the link refers to this rule

**Review by shalithasuranga — COMMENTED** [review]
Please check inline comments @ChathuminaVimukthi

File: src/rulesets/obsoleteAttr.js (hunk 2 above)
Comment (ChathuminaVimukthi) [reply src/rulesets/obsoleteAttr.js]: https://www.tutorialspoint.com/html5/html5_deprecated_tags.htm

Please refer the Deprecated Attributes column. I use this site for my academic purposes :)

Comment (shalithasuranga) [reply src/rulesets/obsoleteAttr.js]: @ChathuminaVimukthi I was unable to find `pubdate` attribute(Even `time` tag) from above page on TP. Can you point out where exactly the rule came from.. Thanks

Comment (ChathuminaVimukthi) [reply src/rulesets/obsoleteAttr.js]: @shalithasuranga  Please refer this article on https://www.sitepoint.com/5-obsolete-features-html5/ 
Thanks

Comment (shalithasuranga) [reply src/rulesets/obsoleteAttr.js]: @ChathuminaVimukthi  it is very old article. I cannot find `pubdate` attribute from here https://www.w3.org/TR/html5/obsolete.html (The html5 reference for obsolete attributes)

Comment (shalithasuranga) [reply src/rulesets/obsoleteAttr.js]: You are free to pickup new obsolete attribute from https://www.w3.org/TR/html5/obsolete.html and update your branch I guess @ChathuminaVimukthi Better to use the w3c official sources . Thanks

Comment (ChathuminaVimukthi) [reply src/rulesets/obsoleteAttr.js]: @shalithasuranga I'm sorry for the mess. I didn't refer the w3. May I change the commits?

Comment (shalithasuranga) [reply src/rulesets/obsoleteAttr.js]: No worries @ChathuminaVimukthi  You can add new valid rule from w3c and update this

Comment (ChathuminaVimukthi) [reply src/rulesets/obsoleteAttr.js]: @shalithasuranga I updated the rules :) Thanks for the advices

**Review by shalithasuranga — APPROVED** [review]
Well done 👍

**Comment by ChathuminaVimukthi** [comment]
👍 Thank you

**Pull request merged**